### PR TITLE
Replace custom commands with @10up/cypress-wp-utils

### LINKED ISSUE #161: Replace custom commands with @10up/cypress-wp-utils
### Is your enhancement related to a problem? Please describe.

All custom commands introduced in this repo have been [migrated](https://github.com/10up/cypress-wp-utils/issues/5) to @10up/cypress-wp-utils. So it's time to use the library for this repo.

### Designs

_No response_

### Describe alternatives you've considered

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by dinhtungdu**
### Description of the Change
This PR updates the Cypress test suite to use `@10up/cypress-wp-utils` instead of custom commands. This is currently blocked by https://github.com/10up/cypress-wp-utils/pull/51.

Closes #161

### Verification Process
See tests pass on both local and GH Actions.

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my change.
- [x] All new and existing tests passed.

### Changelog Entry

> Changed - Replaced custom commands with @10up/cypress-wp-utils in end-to-end tests

### Credits

Props @dinhtungdu

### CODE REVIEW

File: tests/cypress/integration/admin.test.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
-		cy.get( '#deactivate-simple-podcasting' ).should( 'be.visible' );
-	} );
-} );
+describe('Admin can login and make sure plugin is activated', () => {
+	before(() => {
+		cy.login();
+	});
+
+	it('Can activate plugin if it is deactivated', () => {
+		cy.visit('/wp-admin/plugins.php');
```
Comment (cadic) [inline tests/cypress/integration/admin.test.js]: Could we use `cy.activatePlugin()` and `cy.deactivatePlugin()` here?

File: tests/cypress/integration/taxonomy.test.js [hunk 2]
```diff
@@ -1,87 +1,76 @@
...
 			'Once at least one podcast exists'
 		);
-	} );
+	});
 
-	it( 'Can delete all taxonomies', () => {
-		cy.visitAdminPage(
-			'edit-tags.php?taxonomy=podcasting_podcasts&podcasts=true'
+	it('Can delete all taxonomies', () => {
+		cy.visit(
```
Comment (cadic) [inline tests/cypress/integration/taxonomy.test.js]: Could we shortcut this with `cy.deleteAllTerms('podcasting_podcasts')`?

**Review by cadic — CHANGES_REQUESTED** [review]
@dinhtungdu thank you for this great update!

I left couple questions about other commands we can potentially replace.

**Comment by dinhtungdu** [comment]
@cadic Thanks for the review. I updated the PR per your suggestion. Please take another look!

**Review by cadic — APPROVED** [review]
LGTM, thank you @dinhtungdu!

**Pull request merged**